www_src/components/el/types/link.jsx

```diff
@@ -47,6 +58,14 @@ var Link = React.createClass({
     });
   },
 
+  onClick: function () {
+    if (this.state.editing) {
+      this.activate();
+    } else {
+      dispatcher.fire('linkClicked', this.props);
```

Comment:
It allows this event's actions to be decoupled...basically I put the business/UI logic at a higher level.